doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -345,6 +345,13 @@ Modem libraries
     * Removed dependency on the :ref:`lib_modem_jwt` library.
       The :ref:`lib_location` library now selects :kconfig:option:`CONFIG_NRF_CLOUD_REST_AUTOGEN_JWT` when using :kconfig:option:`CONFIG_NRF_CLOUD_REST`.
 
+* :ref:`lib_azure_iot_hub` library:
+
+  * Added:
+
+    * :kconfig:option:`CONFIG_AZURE_IOT_HUB_NATIVE_TLS` option for configuring the socket to be native for TLS instead of offloading
+      TLS operations to the modem.
+
```

Comment:
```suggestion
* :ref:`lib_azure_iot_hub` library:

  * Added :kconfig:option:`CONFIG_AZURE_IOT_HUB_NATIVE_TLS` option to configure the socket to be native for TLS instead of offloading TLS operations to the modem.
```
Should be below the "Libraries for networking" heading.